docs/commands/rpc-protocol.md

```diff
@@ -1700,6 +1700,86 @@ Removing node ID (restart required to take effect)
 
 ---
 
+### node_telemetry
+_version 21.0+_  
+Return metrics from nodes.   
+**Request:**
+```json
+{
+  "action": "node_telemetry"
+}
+```
+**Response:**
+```json
+{
+    "block_count": "5777903",
+    "cemented_count": "688819",
+    "unchecked_count": "443468",
+    "account_count": "620750",
+    "bandwidth_cap": "1572864",
+    "peer_count": "32",
+    "protocol_version": "18",
+    "uptime": "556896",
+    "genesis_block": "F824C697633FAB78B703D75189B7A7E18DA438A2ED5FFE7495F02F681CD56D41",
+    "major_version": "21",
+    "minor_version": "0",
+    "patch_version": "0",
+    "pre_release_version": "0",
+    "maker": "0",
+    "timestamp": "1587055945990"
+}
+```
+This contains a summarized view of the network, 10% of lower/upper bound results are removed to reduce the effect of outliers. Most values returned are averages, in some cases the mode (most common) is returned such as the **genesis_block** & **\*_version/maker** fields. **bandwidth_cap** can return either, 0 has a special meaning of unlimited, so the mode is chosen if there is more than 1 common result otherwise the results are averaged (excluding 0). **maker** is meant for third party node software so that it can be distinguished, **0** represents the Nano Foundation. **uptime** is measure in seconds and **timestamp** is the number of milliseconds since the UTC epoch at the point where the repsonse is sent from the peer.
+
+This only returns values which have been cached by the ongoing polling of peer metric data. Each response is cached for 15 seconds on beta and 60 seconds on the main network; there is also an additional few seconds added on for response delays.
```

Comment:
Consider noting the cache time of the main network first.

Reply:
Done